src/i18n/de.json

```diff
@@ -65,21 +66,23 @@
     },
     "tabs": {
       "overview": "Studie",
-      "data": "Monitoring",
       "participants": "Teilnehmer:innen",
       "observations": "Datenerhebungen",
-      "interventions": "Interventionen",
       "integration": "Integrationen",
-      "timeline": "Kalender"
+      "interventions": "Interventionen",
+      "timeline": "Kalender",
+      "monitoring": "Monitoring",
+      "data": "Daten"
     },
     "tabLink": {
       "overview": "Overview",
-      "data": "Monitoring",
       "participants": "Participants",
       "observations": "Observations",
-      "interventions": "Interventions",
       "integration": "Integrations",
-      "timeline": "Timeline"
+      "interventions": "Interventions",
+      "timeline": "Timeline",
+      "monitoring": "Monitoring",
+      "data": "Data"
```

Comment:
🇺🇸/🇬🇧 vs. 🇩🇪/🇦🇹

Reply:
These translations were used for the tab links (vue-routes), but good point, we do not need a translation for these links. I'll remove the translations and use the route name directly.